airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/spec.json

```diff
@@ -0,0 +1,60 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Yahoo Finance Spec",
+    "type": "object",
+    "required": [
+      "tickers"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "tickers": {
+        "type": "string",
```

Comment:
I have decided to use a string of comma-separated values because one may want to retrieve data from **a lot** of different securities at once, and the current state of `array` input on the web view does not allow bulk-paste of a list.